extensions/vertx/deployment/src/test/java/io/quarkus/vertx/EventBusCodecTest.java

```diff
@@ -73,6 +77,11 @@ String getMessage() {
         }
     }
 
+    @Retention(RetentionPolicy.CLASS)
```

Comment:
Is this required?

Reply:
Not really, class retention is the default. I just wanted to be explicitly close to the original reproducer.

Reply:
It could be a runtime-retained annotation as well, that wouldn't make a difference.